src/App.js

```diff
@@ -12,22 +11,18 @@ const App = () => (
       <nav>
         <ul>
           <li>
-            <Link to="/">Home</Link>
+            <Link to="/">Books</Link>
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, in the components folder, you should have a Books component. At the moment that is missing. See screenshots below describing exactly what you need to add.
- Review guideline
![image](https://github.com/mpa-mxiang/bookstores/assets/62849872/d33926ba-8b6f-4e16-9790-3e5471b4c1d7)
- Project requirements
![image](https://github.com/mpa-mxiang/bookstores/assets/62849872/9bb91811-bc1c-475b-940f-f006a26a3aa4)

Reply:
- [ ] I think you misunderstood what the previous reviewer tried to say here, You only renamed the `Home` component to the `Books` component.
As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md#:~:text=/categories-,Create%20components%20inside%20of%20the%20/components%20directory,-%3A), Please do the following:
- Put the Books component inside the `src/components` directory and you also need to create 3 other components and render them inside the `Books` component:
1. `Book` component: Takes the book data as a prop and renders the title and the author and Has a button for removing books.
2. `BookList` component: Renders a list of books using the `Book` component.
3. `NewBookForm` component: Renders a form with inputs for the title and the author of the book and Has a button to submit.